pcc/src/constants.ts

```diff
@@ -0,0 +1,7 @@
+import os from 'os';
+import path from 'path';
+
+export const PCC_ROOT_DIR = path.join(os.homedir(), '.pcc');
+export const GOOGLE_CLIENT_ID =
+  '142470191541-8o14j77pvagisc66s48kl4ub91f9c7b8.apps.googleusercontent.com';
+export const GOOGLE_REDIRECT_URI = 'http://localhost:3030/oauth-redirect';
```

Comment:
I thought that Google stopped allowing localhost redirect urls?
https://developers.google.com/identity/protocols/oauth2/resources/loopback-migration

Extra reasoning
https://stackoverflow.com/questions/67343399/why-is-it-acceptable-to-use-the-http-scheme-for-oauth-loopback-interface-redir